Makefile

```
@@ -7,7 +7,7 @@ STYLUA   ?= $(shell which stylua 2>/dev/null || echo "$(HOME)/.local/share/nvim/
 PROJECT ?= lua/ tests/
 LOGDIR  ?= .luals-log
 
-.PHONY: luals selene selene-file format-check format format-file check test validate install-hooks
+.PHONY: luals selene selene-file format-check format format-file check test validate benchmark-tool-call-folding install-hooks
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)